oap-server/server-library/library-client/src/test/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ITElasticSearchClient.java

```diff
@@ -183,4 +183,14 @@ public void bulk() throws InterruptedException {
         bulkProcessor.flush();
         bulkProcessor.awaitClose(2, TimeUnit.SECONDS);
     }
+
+    @Test
+    public void undoFormatIndexName() {
```

Comment:
This is an IT test, Integration Test. You should test `retrievalIndexByAliases` with real Elastic, create indexes with namespace, then test `#retrievalIndexByAliases` result

Reply:
Take an example with other tests in this class.